.github/workflows/build_gcp_azure_manual.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       PKR_VAR_gallery_resource_group: rg-worker_images-public-westeurope
       PKR_VAR_gallery_name: spacelift_worker_images_public
       PKR_VAR_gallery_image_name: spacelift_worker_image
-      PKR_VAR_gallery_replication_regions: '["westeurope"]'
+      PKR_VAR_gallery_replication_regions: '["westeurope", "northeurope", "northcentralus", "eastus", "eastus2", "westus2", "westus3"]'
```

Comment:
I think the way you've written the packer code this will result in a duplicate `target_region` block for `westeurope` because you explicitly create one for `var.location`, and then create one for each of these regions.

Reply:
@adamconnelly pushed a fix. plus `var.location` is null anyway, so wouldn't have worked.

Reply:
It must be getting the location from the resource group right now or something along those lines.

Reply:
probably. should I remove the variable?

Reply:
removed